code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -66,6 +66,15 @@
 	ammo_x_offset = 4
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/gun/energy/e_gun/blueshield
+	name = "advanced stun revolver"
+	desc = "An advanced stun revolver with the capacity to shoot both electrodes and lasers."
+	icon_state = "bsgun"
+	item_state = "gun"
+	force = 7
+	ammo_type = list(/obj/item/ammo_casing/energy/electrode/hos, /obj/item/ammo_casing/energy/laser/hos)
```

Comment:
I don't like the idea of something that shoots lasers or tasers, tasers kind of are terribly unfun when it comes to combat and are used on HOS gun only for a reason